blackduck-artifactory-common/src/main/java/com/synopsys/integration/blackduck/artifactory/modules/scaaas/ScanAsAServiceModule.java

```diff
@@ -58,4 +77,72 @@ public void handleBeforeDownloadEvent(Request request, RepoPath repoPath) {
             logger.info(String.format("BlackDuck Header present; Allowing download: repo: %s", repoPath));
         }
     }
+
+    public String deleteScanAsAServicePropertiesOnRepos(Map<String, List<String>> params) {
+        // A list pf repos should be provided in the params. If no repos provided, send error message
+        if (!params.containsKey("repos")) {
+            return "Unable to execute. Please check that a list of repos is provided.";
+        }
+        List<String> configuredRepos = scanAsAServiceModuleConfig.getBlockingRepos();
+        List<String> requestedRepos = params.get("repos");
+        List<String> reposToCheck = requestedRepos.stream().filter(configuredRepos::contains).collect(Collectors.toList());
```

Comment:
Do we need to filter by configured repos?  If a customer changed the configured repos and forgot to clean properties then, they may still want to include them in this command.

Reply:
Fair point. I will exclude that check.